std/haxe/io/Bytes.hx

```diff
@@ -111,6 +120,9 @@ class Bytes {
 		#end
 	}
 
+	/**
+	  Sets the value of this instance's `len` bytes to `value`, starting from index `pos`.
```

Comment:
"Sets `len` consecutive bytes starting from index `pos` of `this` instance to `value`."